.travis.yml

```diff
@@ -1,31 +1,15 @@
-dist: trusty
```

Comment:
Can we run these in GH actions? We can tackle that in a separate PR.

Reply:
yeah there are already more changes than i would have liked in this PR 😅